src/models.py

```diff
@@ -45,6 +45,7 @@
     NullPhraseLayer
 from .modules.edge_probing import EdgeClassifierModule
 from .modules.seq2seq_decoder import Seq2SeqDecoder
+from .bert.utils import BertEmbedderModule
```

Comment:
Do lazy-import as in the existing implementation: https://github.com/jsalt18-sentence-repl/jiant/blob/master/src/models.py#L147